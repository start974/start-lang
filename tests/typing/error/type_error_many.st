def test1 : N2 := 1
def test2 : N := 1
def test3 : N3 := 1
def test4 : N3 := 1
