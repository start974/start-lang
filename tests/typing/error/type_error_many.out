Error[301]: tests/typing/error/type_error_many.st:1:13
1 | def test1 : N2 := 1
                ^^ Type 'N2' not found

Error[301]: tests/typing/error/type_error_many.st:3:13
3 | def test3 : N3 := 1
                ^^ Type 'N3' not found

Error[301]: tests/typing/error/type_error_many.st:4:13
4 | def test4 : N3 := 1
                ^^ Type 'N3' not found
